.agents/rules/commit-convention.md

```
@@ -0,0 +1,30 @@
+---
+name: commit-convention
+description: Use when creating git commits. Enforces Conventional Commits format with lowercase types and scopes.
+---
+
+# Commit Convention
+
+Conventional Commits (enforced by husky + commitlint).
+
+## Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)